src/observer_cli_lib.erl

```diff
@@ -362,6 +362,9 @@ parse_cmd(ViewOpts, Module, Args) ->
             hide;
         "`\n" ->
             scheduler_usage;
+        [$p, $i, $d | PidStr] ->
```

Comment:
looks great! 
If we support pid:<0.1.0>, 
maybe we should also support 
name:atom_name
name:{global,register_name} ?